task/06-conditions-n-loops-tasks.js

```diff
@@ -192,7 +210,23 @@ function findFirstSingleChar(str) {
  *
  */
 function getIntervalString(a, b, isStartIncluded, isEndIncluded) {
-    throw new Error('Not implemented');
+    if (a < b) {
+        if (isStartIncluded == true && isEndIncluded == true) {
+            return `[${a}, ${b}]`;
+        } else if (isStartIncluded == false && isEndIncluded == true) {
+            return `(${a}, ${b}]`;
+        } else if (isStartIncluded == true && isEndIncluded == false) {
+            return `[${a}, ${b})`;
+        } else return `(${a}, ${b})`;
+    } else {
+        if (isStartIncluded == true && isEndIncluded == true) {
+            return `[${b}, ${a}]`;
+        } else if (isStartIncluded == false && isEndIncluded == true) {
+            return `(${b}, ${a}]`;
+        } else if (isStartIncluded == true && isEndIncluded == false) {
```

Comment:
`if (isStartIncluded === true)` is the same like  `if(isStartIncluded)`
`if (isEndIncluded === false)` is the same like  if(!isEndIncluded)
Please, see https://javascript.info/logical-operators

Please, rework the code